src/pages/Fox/components/FoxWifHatClaim/FoxWifHatClaim.tsx

```diff
@@ -0,0 +1,108 @@
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { useQueryClient } from '@tanstack/react-query'
+import { AnimatePresence } from 'framer-motion'
+import { lazy, Suspense, useCallback, useMemo, useState } from 'react'
+import { MemoryRouter, Route, Switch, useLocation } from 'react-router'
+import { makeSuspenseful } from 'utils/makeSuspenseful'
+import { getFoxWifHatClaimedQueryKey } from 'pages/Fox/hooks/useFoxWifHatClaimed'
+import { useFoxWifHatMerkleTreeQuery } from 'pages/Fox/hooks/useFoxWifHatMerkleTreeQuery'
+
+import type { FoxWifHatClaimRouteProps } from './types'
+import { FoxWifHatClaimRoutePaths } from './types'
+
+const suspenseFallback = <div>Loading...</div>
+
+const defaultBoxSpinnerStyle = {
+  height: '500px',
+}
+
+const FoxWifHatClaimConfirm = makeSuspenseful(
+  lazy(() =>
+    import('./FoxWifHatClaimConfirm').then(({ FoxWifHatClaimConfirm }) => ({
+      default: FoxWifHatClaimConfirm,
+    })),
+  ),
+  defaultBoxSpinnerStyle,
+)
+
+const FoxWifHatClaimStatus = makeSuspenseful(
+  lazy(() =>
+    import('./FoxWifHatClaimStatus').then(({ FoxWifHatClaimStatus }) => ({
+      default: FoxWifHatClaimStatus,
+    })),
+  ),
+  defaultBoxSpinnerStyle,
+)
+
+const FoxWifHatClaimEntries = [FoxWifHatClaimRoutePaths.Confirm, FoxWifHatClaimRoutePaths.Status]
+
+export const FoxWifHatClaim: React.FC<FoxWifHatClaimRouteProps> = ({ accountId }) => {
+  return (
+    <MemoryRouter initialEntries={FoxWifHatClaimEntries} initialIndex={0}>
+      <FoxWifHatClaimRoutes accountId={accountId} />
+    </MemoryRouter>
+  )
+}
+
+export const FoxWifHatClaimRoutes: React.FC<FoxWifHatClaimRouteProps> = ({ accountId }) => {
+  const location = useLocation()
+  const queryClient = useQueryClient()
+  const getFoxWifHatClaimsQuery = useFoxWifHatMerkleTreeQuery()
+
+  const claimQuote = useMemo(() => {
```

Comment:
vernacular nitpick across this PR: Not sure about the `quote` vernacular.
As far as magic internet money state machine goes, it *is* an available claim, not a quote, i.e should be able to deterministically claim exactly that from the contract.

Woule prefer to simply use `claims` here, or `available claims` if we feel like it, but at the end of the day happy with any of the options, just naming OCD haha

Reply:
Done in [7e9a198](https://github.com/shapeshift/web/pull/8611/commits/7e9a198a552c9ffbe24675fdb83a91d1875e342b)